### ISSUE TITLE
nvda seems to lag after a thunderbird dialogue came up asking if I want to retry logging in.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1706]
**Reported by dwillemv on 2011-08-05 07:26**
in v 4576:
I noticed nvda being unable to focus the thunderbird window, so I restarted nvda, but I had the same problem after the restart. 
a snip from the log
INFO - core.main (09:06:14):
Using wx version 2.8.11.0 (msw-unicode)
INFO - braille.initialize (09:06:14):
Using liblouis version 2.3.0
INFO - braille.BrailleHandler.setDisplayByName (09:06:14):
Loaded braille display driver noBraille
INFO - core.main (09:06:21):
NVDA initialized
DEBUGWARNING - eventHandler.executeEvent (09:08:49):
error executing event: documentLoadComplete on <NVDAObjects.Dynamic_DocumentBrokenFocusedStateGecko1_9MozillaIAccessibleWindowNVDAObject object at 0x065F3AF0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 133, in executeEvent
  File "eventHandler.pyc", line 174, in doPreDocumentLoadComplete
  File "treeInterceptorHandler.pyc", line 17, in getTreeInterceptor
  File "virtualBuffers\gecko_ia2.pyc", line 85, in **contains**
COMError: (-2147418110, 'Call was canceled by the message filter.', (None, None, None, 0, None))
ERROR - scriptHandler.executeScript (09:08:49):
error executing script: <bound method GlobalCommands.script_reportCurrentLine of <globalCommands.GlobalCommands object at 0x0649DD90>> with gesture u'NVDA+up arrow'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 165, in executeScript
  File "globalCommands.pyc", line 63, in script_reportCurrentLine
  File "virtualBuffers__init__.pyc", line 579, in makeTextInfo
  File "virtualBuffers__init__.pyc", line 86, in **init**
  File "textInfos\offsets.pyc", line 260, in **init**
  File "virtualBuffers__init__.pyc", line 98, in _getCaretOffset
  File "virtualBuffers__init__.pyc", line 91, in _getSelectionOffsets
WindowsError: [1775](Error) A null context handle was passed from the client to the host during a remote procedure call
ERROR - scriptHandler.executeScript (09:08:49):
error executing script: <bound method GlobalCommands.script_reportCurrentLine of <globalCommands.GlobalCommands object at 0x0649DD90>> with gesture u'NVDA+up arrow'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 165, in executeScript
  File "globalCommands.pyc", line 63, in script_reportCurrentLine
  File "virtualBuffers__init__.pyc", line 579, in makeTextInfo
  File "virtualBuffers__init__.pyc", line 86, in **init**
  File "textInfos\offsets.pyc", line 260, in **init**
  File "virtualBuffers__init__.pyc", line 98, in _getCaretOffset
  File "virtualBuffers__init__.pyc", line 91, in _getSelectionOffsets
WindowsError: [1775](Error) A null context handle was passed from the client to the host during a remote procedure call

**Comment by ehollig** [comment]
CC @leonardder @Brian1Gaff

**Comment by dwillemv** [comment]
I honestly have no idea. I have stopped using thunderbird long ago.
I'd guess after 7 years and many many other users it was either fixed
in another issue or had something to do with the setup I had at the
time. I'd say close it and if it really is a problem someone can log a
new issue.

**Comment by ehollig** [comment]
Closing as requested in https://github.com/nvaccess/nvda/issues/1706#issuecomment-408166768. If this really is a problem, please create a new issue with exact steps to reproduce.
